app/components/sidebars/main/main_sidebar.js

```diff
@@ -45,7 +45,6 @@ export default class ChannelSidebar extends Component {
         currentUserId: PropTypes.string.isRequired,
         teamsCount: PropTypes.number.isRequired,
         theme: PropTypes.object.isRequired,
-        previewChannel: PropTypes.func,
```

Comment:
This prop is being passed from the Channel screen, we should remove that as well

Reply:
Yes.... `previewChannel` was being passed all through the component tree across several key components. Handled in commit.